prow/githuboauth/githuboauth.go

```diff
@@ -92,6 +109,8 @@ func NewAgent(config *config.GitHubOAuthConfig, logger *logrus.Entry) *Agent {
 // redirect user to GitHub OAuth end-point for authentication.
 func (ga *Agent) HandleLogin(client OAuthClient) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		destPage := r.URL.Query().Get("dest")
+		rerunStatus := r.URL.Query().Get("rerun_status")
```

Comment:
This feels very specific to the particular reason we are logging in. Would it make sense to be more generic?

Reply:
In the future, this will contain the name of the prow job, so when the page reloads after logging in, the same popup for the same job appears. I renamed it to make it make a little more sense.